GitUI/UserControls/FileStatusList.cs

```diff
@@ -1269,11 +1258,9 @@ int GetWidth()
             }
         }
 
-        private void HandleVisibility_NoFilesLabel_FilterComboBox(bool filesPresent)
+        private void HandleVisibility_ComboBoxes(bool filesPresent)
```

Comment:
Do we need this method at all? It delegates straight to `SetFileStatusListViewBounds`

Reply:
If the label `LoadingFiles` is removed,  HandleVisibility_ComboBoxes will be removed (or if a delay is introduced), otherwise probably kept.
Need a decision on the label first.

Reply:
updated the LoadingFiles label handling